### PR TITLE
Added native mobile app documentation.

### PR DESCRIPTION
**Pull request by brianorwhatever**
@BigBlueHat Starts of the native docs

### CODE REVIEW

File: developers/wallets/index.md [hunk 1]
```diff
@@ -6,18 +6,26 @@ permalink: /developers/wallets/
...
+## Resources
 - **Example Code**: the [chapi-demo-wallet](https://github.com/credential-handler/chapi-demo-wallet) contains a full example implementation and is referenced throughout this guide.
 - **Polyfill Library**: The [credential-handler-polyfill](https://github.com/credential-handler/credential-handler-polyfill) library provides the needed functionality in the browser.
 - **Helper Library**: The [web-credential-handler](https://github.com/credential-handler/web-credential-handler) library provides helper functions for CHAPI integration in your code.
 
-### First, set up your wallet to register itself with the browser as a Credential Handler.
-#### 1. Import the CHAPI Polyfill into your wallet app
+## Wallet Registration
+
+Wallets need to be registered with the browser as a Credential Handler in order to store or retrieve Verifiable Credentials.
```
Comment (dlongley) [inline developers/wallets/index.md:22]:
```suggestion
Wallets need to be registered with the browser as a Credential Handler in order to store or retrieve Verifiable Credentials on the Web.
```

File: developers/wallets/index.md [hunk 2]
```diff
@@ -68,11 +76,8 @@ You can register a Credential Handler by calling the `CredentialManager.requestP
   target="_blank" rel="noopener noreferrer"> chapi-demo-wallet/install-wallet.js </a>
 </p>
 
-### Then, configure your wallet to respond to Credential Handler events.
-Your wallet app's existing functionality can be configured to respond to CHAPI events.
-
-#### 4. Setup Listeners for CHAPI Events
-The `activateHandler()` function is a helper that sets up listeners for CHAPI `get()` and `store()` events.
+### 4. Setup Listeners for CHAPI Events
+Configure your wallet to respond to Credential Handler events. The `activateHandler()` function is a helper that sets up listeners for CHAPI `get()` and `store()` events.
```
Comment (dlongley) [inline developers/wallets/index.md:80]: We should be clear that there are two ways to receive requests from CHAPI, one is via CHAPI events and the other via URLs (which can be deep / "app" links).

File: developers/wallets/index.md [hunk 3]
```diff
@@ -91,27 +96,87 @@ WebCredentialHandler.activateHandler({
...
-  async function handleGetEvent() {
-    const event = await WebCredentialHandler.receiveCredentialEvent();
+{% endtab %}
 
-    console.log('Wallet processing get() event:', event);
+{% tab wallet for native mobile wallets %}
 
-    //Your wallet's code for responding to a request for a Verifiable Credential
+To enable a native mobile wallet to receive VCs via CHAPI, your application will need to be able to handle deeplinks. See appropriate deep link documentation for [Android](https://developer.android.com/training/app-links/deep-linking) or [iOS](https://developer.apple.com/documentation/xcode/allowing-apps-and-websites-to-link-to-your-content?language=objc).
+
```
Comment (dlongley) [inline developers/wallets/index.md:105]:
```suggestion
To enable a native mobile wallet to receive VCs via CHAPI, your application will need to be able to handle deep links. Deep links, also known as app links, cause a user's Web browser to open a native application and pass the URL to it when the link is followed. See appropriate deep link documentation for [Android](https://developer.android.com/training/app-links/deep-linking) or [iOS](https://developer.apple.com/documentation/xcode/allowing-apps-and-websites-to-link-to-your-content?language=objc).

```

File: developers/wallets/index.md (hunk 1 above)
Comment (BigBlueHat) [reply developers/wallets/index.md:22]: Resolved in forthcoming commit.

File: developers/wallets/index.md (hunk 3 above)
Comment (BigBlueHat) [reply developers/wallets/index.md:105]: Resolved in upcoming commit.

**Comment by BigBlueHat** [comment]
Closing in favor of https://github.com/credential-handler/chapi.io/pull/27

**Pull request closed without merging**